tests/TestCase/Model/Behavior/ProfferBehaviorTest.php

```diff
@@ -157,6 +157,10 @@ public function beforeMarshalProvider()
 
     /**
      * @dataProvider beforeMarshalProvider
+     *
+     * @param array $data
```

Comment:
if possible, be specific about types (e.g. if it's an array of strings, use `string[]` instead of just `array`, but not always possible)